src/common/popup/components/error-details.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import StackTrace from 'stacktrace-js';
+
+import CopyButton from './copy-button';
+
+import ErrorShape from '../utils/error-shape';
+
+class ErrorDetails extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      stackframes: null,
+    };
+  }
+
+  componentDidMount() {
+    const { error } = this.props;
+
+    StackTrace.fromError(error)
+      .then(stackframes => this.setState({ stackframes }))
+      .catch(err => console.error(err));
```

Comment:
👍 for outputting it on the console still :)

Reply:
This is only logging an error in case the stacktrace enhancement (apply source maps) fails.